_source/user-guide/accounts/pci-dds.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: article
+title: PCI compliance and log redaction
+permalink: /user-guide/accounts/pci-compliance.html
+flags:
+  admin: true
+  logzio-plan: enterprise
+tags:
+  - accounts
+contributors:
+  - shalper
+---
+
+Minimize the risk of exposing sensitive customer data with Logz.io's log redaction function and benefit from a logging management solution that offers
+automated sensitive-data scrubbing.
+
+If your logs need to be scrutinized for sensitive PPI such as customer credit card data, you can enable the PCI compliance function on your Logz.io accounts. Event logs are no exception when it comes to PCI compliance and Logz.io makes it simpler to prevent logging vulnerabilities and minimize the possibility of data breaches in your raw log files.
+
+When enabled, logs will be subject to an automated regex search and scrubbed of any sensitive customer data before they are indexed by Logz.io. The process occurs before the data is processed by the ElasticSearch database and does not affect data archiving.
+
+Logz.io's PCI compliance solution is compatible with all merchant levels,
+including the most stringent PCI Compliance Level 1.
+
+For more information on PCI compliance and DDS (Data Security Standard), see the PCI Security Standards Council's [resources](https://www.pcisecuritystandards.org/).
+{:.info-box.read}
+
+#### To enable PCI compliance
+
+<div class="tasklist">
+
+##### Select an account
+
+You can control PCI compliance from [**<i class="li li-gear"></i> > Settings > General**](https://app.logz.io/#/dashboard/settings/general) in the top menu.
+
+![Account settings](https://dytvr9ot2sszz.cloudfront.net/logz-docs/accounts/pci.png)
+
+If you have more than one account or a combination of sub accounts, repeat this process for each of them.
+
+##### Request activation and accept the terms
+
+Once you accept the legal terms and click `activate`, Logz.io Support will be automatically notified and your ticket will be created.
+
+##### Vetting the regex pattern
+
+Support will walk you through the steps of creating an appropriate regex pattern and specifying which fields to apply it to. The process is sensitive and care must be taken to minimize its effects as it can cause ingestion latency.
```

Comment:
This is actually the opposite of how the process works; the customer must be the one to provide us with the regex pattern and the field/s to which it needs to be applied, as any inaccuracies might cause a breach in compliance - not something we want to be liable for.

We might reach out to the customer for more details if we see something is way off with the pattern, but we cannot be the ones writing it.